content/en/docs/contrib/writing-code.md

```diff
@@ -0,0 +1,162 @@
+start.
+
+Those of you who are unfamiliar with `git` and GitHub will find that this tutorial and its links
+include just enough information to get started. However, you'll probably want to read some more
+about these different tools if you plan on contributing to SpinKube regularly.
+
+For the most part though, this tutorial tries to explain as much as possible, so that it can be of
+use to the widest audience.
+
+## Code of Conduct
+
+As a contributor, you can help us keep the SpinKube community open and inclusive. Please read and
+follow our [Code of Conduct](https://www.fermyon.com/code-of-conduct).
+
+## Install git
+
+For this tutorial, you'll need Git installed to download the current development version of SpinKube
+and to generate a branch for the changes you make.
+
+To check whether or not you have Git installed, enter `git` into the command line. If you get
+messages saying that this command could not be found, you'll have to download and install it. See
+[Git's download page](https://git-scm.com/download) for more information.
+
+If you're not that familiar with Git, you can always find out more about its commands (once it's
+installed) by typing `git help` into the command line.
+
+## Fork the repository
+
+SpinKube is hosted on GitHub, and you'll need a GitHub account to contribute. If you don't have one,
+you can sign up for free at [GitHub](https://github.com).
+
+SpinKube's repositories are organized under the [spinkube GitHub
+organization](https://github.com/spinkube). Once you have an account, fork one of the repositories
+by visiting the repository's page and clicking "Fork" in the upper right corner.
+
+Then, from the command line, clone your fork of the repository. For example, if you forked the
+`spin-operator` repository, you would run:
+
+```shell
+git clone https://github.com/YOUR-USERNAME/spin-operator.git
+```
+
+## Read the README
+
+Each repository in the SpinKube organization has a README file that explains what the project does
+and how to get started. This is a great place to start, as it will give you an overview of the
+project and how to run the test suite.
+
+## Run the test suite
+
+When contributing to a project, it's very important that your code changes don't introduce bugs. One
+way to check that the project still works after you make your changes is by running the project's
+test suite. If all the tests still pass, then you can be reasonably sure that your changes work and
+haven't broken other parts of the project. If you've never run the project's test suite before, it's
+a good idea to run it once beforehand to get familiar with its output.
+
+Most projects have a command to run the test suite. This is usually something like `make test` or
+`cargo test`. Check the project's README file for instructions on how to run the test suite. If
+you're not sure, you can always ask for help in the `#spinkube` channel on Slack.
```

Comment:
Worth another link to the channel here?

Reply:
Yeah, it was referenced quite a while back in the document.

Reply:
done